client/js/views/GameView.vue

```diff
@@ -711,7 +711,7 @@ export default {
       this.$nextTick(function () {
         const container = this.$refs.logsContainer;
         if (container) {
-          container.scrollTop = container.scrollHeight + 120;
+          container.scrollTop = container.scrollHeight;
```

Comment:
This is the right idea, but because this functionality overlaps so much with your additions on 729-730, let's pull the guts of this watcher into it's own method `scrollToLastLog()` which we then call inside the watcher and inside mounted.

Reply:
ok thanks, i'll fix these right now

Reply:
Ok i created the changes